
Command to create ssh tunnel:
ssh -N -f -L 8001:tiger-h19c1n16:8001 tmarcu@tigergpu.princeton.edu

Use a Browser on your local machine to go to:
localhost:8001  (prefix w/ https:// if using password)

[I 21:30:42.042 LabApp] JupyterLab extension loaded from /home/tmarcu/.conda/envs/brain2brain_env/lib/python3.7/site-packages/jupyterlab
[I 21:30:42.042 LabApp] JupyterLab application directory is /home/tmarcu/.conda/envs/brain2brain_env/share/jupyter/lab
[I 21:30:42.116 LabApp] Serving notebooks from local directory: /home/tmarcu/brain2brain
[I 21:30:42.116 LabApp] The Jupyter Notebook is running at:
[I 21:30:42.116 LabApp] http://tiger-h19c1n16:8001/
[I 21:30:42.116 LabApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[W 21:31:19.629 LabApp] Could not determine jupyterlab build status without nodejs
[I 21:31:25.073 LabApp] Kernel started: 0979d59f-f7fb-4c00-be0d-b311a16805cb
[I 21:31:25.100 LabApp] Kernel started: f1eab61a-f14c-4a4b-93b0-ce078e94c8f3
[I 21:31:25.124 LabApp] Kernel started: 74acb9ed-afb9-4a20-adaa-3c38427a6326
2020-04-29 21:31:34.253287: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'libcuda.so.1'; dlerror: libcuda.so.1: cannot open shared object file: No such file or directory
2020-04-29 21:31:34.253329: E tensorflow/stream_executor/cuda/cuda_driver.cc:351] failed call to cuInit: UNKNOWN ERROR (303)
2020-04-29 21:31:34.253354: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:156] kernel driver does not appear to be running on this host (tiger-h19c1n16): /proc/driver/nvidia/version does not exist
2020-04-29 21:31:34.253627: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 AVX512F FMA
2020-04-29 21:31:34.261096: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2400000000 Hz
2020-04-29 21:31:34.261241: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x55c562e32d70 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-04-29 21:31:34.261254: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-04-29 21:33:03.450064: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'libcuda.so.1'; dlerror: libcuda.so.1: cannot open shared object file: No such file or directory
2020-04-29 21:33:03.450107: E tensorflow/stream_executor/cuda/cuda_driver.cc:351] failed call to cuInit: UNKNOWN ERROR (303)
2020-04-29 21:33:03.450137: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:156] kernel driver does not appear to be running on this host (tiger-h19c1n16): /proc/driver/nvidia/version does not exist
2020-04-29 21:33:03.450370: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 AVX512F FMA
2020-04-29 21:33:03.459452: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2400000000 Hz
2020-04-29 21:33:03.459606: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x56362e1c1aa0 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-04-29 21:33:03.459621: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
[I 21:33:23.779 LabApp] Saving file at /experiments/notebooks/one_to_one/wavenet.ipynb
[I 21:33:24.551 LabApp] Saving file at /experiments/notebooks/one_to_one/enc_dec_2.ipynb
[I 21:35:25.028 LabApp] Saving file at /experiments/notebooks/one_to_one/wavenet.ipynb
[I 21:35:25.508 LabApp] Saving file at /experiments/notebooks/one_to_one/enc_dec_2.ipynb
[I 21:35:32.707 LabApp] Kernel started: c38333e4-f51e-497b-8bae-20c3c7303db7
2020-04-29 21:37:08.239280: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'libcuda.so.1'; dlerror: libcuda.so.1: cannot open shared object file: No such file or directory
2020-04-29 21:37:08.239338: E tensorflow/stream_executor/cuda/cuda_driver.cc:351] failed call to cuInit: UNKNOWN ERROR (303)
2020-04-29 21:37:08.239366: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:156] kernel driver does not appear to be running on this host (tiger-h19c1n16): /proc/driver/nvidia/version does not exist
2020-04-29 21:37:08.239656: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 AVX512F FMA
2020-04-29 21:37:08.247726: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2400000000 Hz
2020-04-29 21:37:08.247892: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x56053b21b530 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-04-29 21:37:08.247906: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
[I 21:37:26.158 LabApp] Saving file at /experiments/notebooks/one_to_one/wavenet.ipynb
[I 21:37:31.766 LabApp] Saving file at /experiments/notebooks/many_to_one/wavenet_m2o.ipynb
[I 21:38:18.529 LabApp] Saving file at /experiments/notebooks/many_to_one/wavenet_m2o.ipynb
[I 21:40:19.644 LabApp] Saving file at /experiments/notebooks/many_to_one/wavenet_m2o.ipynb
[I 21:42:20.920 LabApp] Saving file at /experiments/notebooks/many_to_one/wavenet_m2o.ipynb
[I 21:44:22.892 LabApp] Saving file at /experiments/notebooks/many_to_one/wavenet_m2o.ipynb
[I 21:45:27.654 LabApp] Saving file at /experiments/notebooks/one_to_one/enc_dec_2.ipynb
[I 22:03:31.618 LabApp] Saving file at /experiments/notebooks/one_to_one/enc_dec_2.ipynb
[I 22:11:36.713 LabApp] Saving file at /experiments/notebooks/one_to_one/enc_dec_2.ipynb
[I 22:19:40.669 LabApp] Saving file at /experiments/notebooks/one_to_one/enc_dec_2.ipynb
[I 22:23:42.434 LabApp] Saving file at /experiments/notebooks/one_to_one/enc_dec_2.ipynb
[W 22:29:56.596 LabApp] WebSocket ping timeout after 119944 ms.
[W 22:29:56.667 LabApp] WebSocket ping timeout after 119978 ms.
[W 22:29:56.674 LabApp] WebSocket ping timeout after 119943 ms.
[W 22:29:56.733 LabApp] WebSocket ping timeout after 119979 ms.
[W 22:29:56.737 LabApp] WebSocket ping timeout after 119932 ms.
[I 22:30:01.601 LabApp] Starting buffering for 0979d59f-f7fb-4c00-be0d-b311a16805cb:bd7ed525-83a0-43a2-a30c-c5f5d44d4a38
[I 22:30:01.675 LabApp] Starting buffering for f1eab61a-f14c-4a4b-93b0-ce078e94c8f3:9a94f862-b2ef-42db-8329-fac66aafc2db
[I 22:30:01.738 LabApp] Starting buffering for 74acb9ed-afb9-4a20-adaa-3c38427a6326:e4795fa8-5aab-4a29-bb95-76787c0923ee
[W 22:30:07.701 LabApp] WebSocket ping timeout after 119940 ms.
[I 22:30:12.707 LabApp] Starting buffering for c38333e4-f51e-497b-8bae-20c3c7303db7:94b58ad8-0b36-46ae-904b-dfdfd66da322
[I 22:31:35.690 LabApp] 302 GET / (10.36.16.7) 0.59ms
[W 22:31:37.810 LabApp] Could not determine jupyterlab build status without nodejs
[I 22:32:10.414 LabApp] Starting buffering for 0979d59f-f7fb-4c00-be0d-b311a16805cb:d5318708-d02c-4653-a736-b8a98904f1cc
[I 22:32:10.891 LabApp] Starting buffering for f1eab61a-f14c-4a4b-93b0-ce078e94c8f3:5bf42d12-1d40-406b-868d-60afb0ca8782
[I 22:32:11.986 LabApp] Starting buffering for 74acb9ed-afb9-4a20-adaa-3c38427a6326:fd0956d6-ed9f-4900-82c1-139e2b0e3639
[I 22:33:42.498 LabApp] Saving file at /experiments/notebooks/many_to_one/wavenet_m2o.ipynb
[I 22:34:18.201 LabApp] Kernel started: 82275483-5694-4f74-a164-4a358a90a20f
[I 22:34:35.654 LabApp] Kernel started: af053a96-2e29-4700-b7ae-5ca8f20b1a98
[I 22:34:41.923 LabApp] Kernel started: 8cbd3a96-5123-45fe-a272-c81891125aec
[I 22:35:43.390 LabApp] Saving file at /experiments/notebooks/many_to_one/wavenet_m2o.ipynb
[I 22:36:53.320 LabApp] Starting buffering for 8cbd3a96-5123-45fe-a272-c81891125aec:bab965d1-2c90-4a79-be7c-ac8247a29c1b
[I 22:36:53.947 LabApp] Starting buffering for af053a96-2e29-4700-b7ae-5ca8f20b1a98:67dc50da-8634-46b4-a4a7-3bb9597520b0
[I 22:36:54.419 LabApp] Starting buffering for 82275483-5694-4f74-a164-4a358a90a20f:e9bc6641-21d2-4099-9fc5-00bdaf722ebc
[I 22:37:44.359 LabApp] Saving file at /experiments/notebooks/many_to_one/wavenet_m2o.ipynb
[I 22:39:14.798 LabApp] Kernel started: 3166675f-e0a4-4451-a156-2fb72e9ef5f4
[I 22:39:36.369 LabApp] Kernel started: 47133af2-f4d3-4e9a-b96c-d38229c0deb1
[I 22:39:46.100 LabApp] Saving file at /experiments/notebooks/many_to_one/wavenet_m2o.ipynb
[I 22:39:50.636 LabApp] Starting buffering for 3166675f-e0a4-4451-a156-2fb72e9ef5f4:a76f90ce-8dc3-4fb6-bbab-0f77948d4cc1
[I 22:39:51.654 LabApp] Starting buffering for 82275483-5694-4f74-a164-4a358a90a20f:4f753d41-b463-4604-b131-e22b89e84802
[I 22:39:52.300 LabApp] Starting buffering for c38333e4-f51e-497b-8bae-20c3c7303db7:e9f1a9a1-485c-4f7f-8527-e3177c8c475e
2020-04-29 22:40:13.737965: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'libcuda.so.1'; dlerror: libcuda.so.1: cannot open shared object file: No such file or directory
2020-04-29 22:40:13.738101: E tensorflow/stream_executor/cuda/cuda_driver.cc:351] failed call to cuInit: UNKNOWN ERROR (303)
2020-04-29 22:40:13.738130: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:156] kernel driver does not appear to be running on this host (tiger-h19c1n16): /proc/driver/nvidia/version does not exist
2020-04-29 22:40:13.738421: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 AVX512F FMA
2020-04-29 22:40:13.744712: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2400000000 Hz
2020-04-29 22:40:13.744910: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x556c2b275090 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-04-29 22:40:13.744925: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
[I 22:41:35.906 LabApp] Saving file at /experiments/notebooks/many_to_one/baseline_nn_m2o.ipynb
[I 22:43:37.248 LabApp] Saving file at /experiments/notebooks/many_to_one/baseline_nn_m2o.ipynb
[I 22:45:11.281 LabApp] Kernel started: 11c77766-c8da-43c0-95a8-08652434da46
[I 22:45:14.413 LabApp] Kernel started: 6f3b5bae-a0fe-444d-9df9-1632fc7b11e0
[I 22:47:14.042 LabApp] Saving file at /experiments/notebooks/one_to_one/One_Electrode_Baseline.ipynb
[I 22:49:14.436 LabApp] Saving file at /experiments/notebooks/one_to_one/One_Electrode_Baseline.ipynb
[I 22:50:53.108 LabApp] Kernel started: 786f5ac8-80d4-4040-afbe-fd9704ccd707
[I 22:51:14.893 LabApp] Saving file at /experiments/notebooks/one_to_one/One_Electrode_Baseline.ipynb
[I 22:53:04.922 LabApp] Kernel started: 7f75037f-0b1e-4701-9213-9bf0a841c103
[I 22:53:42.844 LabApp] Kernel restarted: 74acb9ed-afb9-4a20-adaa-3c38427a6326
2020-04-29 22:53:48.155980: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'libcuda.so.1'; dlerror: libcuda.so.1: cannot open shared object file: No such file or directory
2020-04-29 22:53:48.156023: E tensorflow/stream_executor/cuda/cuda_driver.cc:351] failed call to cuInit: UNKNOWN ERROR (303)
2020-04-29 22:53:48.156051: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:156] kernel driver does not appear to be running on this host (tiger-h19c1n16): /proc/driver/nvidia/version does not exist
2020-04-29 22:53:48.156259: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 AVX512F FMA
2020-04-29 22:53:48.163361: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2400000000 Hz
2020-04-29 22:53:48.163545: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x56260ca6ec40 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-04-29 22:53:48.163560: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
[I 22:55:20.330 LabApp] Saving file at /experiments/notebooks/one_to_one/enc_dec_2.ipynb
[I 22:57:21.327 LabApp] Saving file at /experiments/notebooks/one_to_one/enc_dec_2.ipynb
[I 22:59:21.947 LabApp] Saving file at /experiments/notebooks/one_to_one/enc_dec_2.ipynb
[I 23:01:23.027 LabApp] Saving file at /experiments/notebooks/one_to_one/enc_dec_2.ipynb
[I 23:03:10.694 LabApp] Saving file at /experiments/notebooks/one_to_one/baseline_nn_2.ipynb
2020-04-29 23:03:15.760394: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'libcuda.so.1'; dlerror: libcuda.so.1: cannot open shared object file: No such file or directory
2020-04-29 23:03:15.760575: E tensorflow/stream_executor/cuda/cuda_driver.cc:351] failed call to cuInit: UNKNOWN ERROR (303)
2020-04-29 23:03:15.760608: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:156] kernel driver does not appear to be running on this host (tiger-h19c1n16): /proc/driver/nvidia/version does not exist
2020-04-29 23:03:15.760860: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 AVX512F FMA
2020-04-29 23:03:15.794000: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2400000000 Hz
2020-04-29 23:03:15.794186: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x55640dad6670 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-04-29 23:03:15.794202: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
[I 23:03:23.840 LabApp] Saving file at /experiments/notebooks/one_to_one/enc_dec_2.ipynb
[I 23:05:11.704 LabApp] Saving file at /experiments/notebooks/one_to_one/baseline_nn_2.ipynb
[I 23:05:24.655 LabApp] Saving file at /experiments/notebooks/one_to_one/enc_dec_2.ipynb
[I 23:07:26.506 LabApp] Saving file at /experiments/notebooks/one_to_one/enc_dec_2.ipynb
[I 23:09:29.085 LabApp] Saving file at /experiments/notebooks/one_to_one/enc_dec_2.ipynb
[I 23:23:30.528 LabApp] Saving file at /experiments/notebooks/one_to_one/wavenet.ipynb
[I 23:26:55.160 LabApp] Saving file at /experiments/notebooks/one_to_one/LinReg.ipynb
[I 23:29:17.009 LabApp] Saving file at /experiments/notebooks/one_to_one/One_Electrode_Baseline.ipynb
[I 23:31:57.735 LabApp] Starting buffering for 47133af2-f4d3-4e9a-b96c-d38229c0deb1:10fe500d-f2b6-4afc-8aff-d99b55eb3d68
[I 23:31:57.927 LabApp] Starting buffering for 0979d59f-f7fb-4c00-be0d-b311a16805cb:3256f803-8bcd-4b4a-953e-d70fbf472ebf
[I 23:31:59.742 LabApp] Starting buffering for 11c77766-c8da-43c0-95a8-08652434da46:6a5af390-b27b-4092-9549-9b02efa91a44
[I 23:31:59.768 LabApp] Starting buffering for 7f75037f-0b1e-4701-9213-9bf0a841c103:021dd995-c7fe-4864-86cb-14381e8b90a8
[I 23:31:59.821 LabApp] Starting buffering for 74acb9ed-afb9-4a20-adaa-3c38427a6326:0e96a539-08a2-4193-979d-57cc25945b7c
[I 23:31:59.846 LabApp] Starting buffering for f1eab61a-f14c-4a4b-93b0-ce078e94c8f3:e38bc50e-33aa-49bb-a9aa-f5b81eeccaa4
[I 23:31:59.886 LabApp] Starting buffering for 6f3b5bae-a0fe-444d-9df9-1632fc7b11e0:d90e8725-c403-447b-9642-3ca8cf187ed9
[I 23:32:02.585 LabApp] Starting buffering for 786f5ac8-80d4-4040-afbe-fd9704ccd707:211ff638-7bdb-4e2b-bcdd-dc0b741e29db
2020-04-29 23:35:07.972371: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'libcuda.so.1'; dlerror: libcuda.so.1: cannot open shared object file: No such file or directory
2020-04-29 23:35:07.972415: E tensorflow/stream_executor/cuda/cuda_driver.cc:351] failed call to cuInit: UNKNOWN ERROR (303)
2020-04-29 23:35:07.972444: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:156] kernel driver does not appear to be running on this host (tiger-h19c1n16): /proc/driver/nvidia/version does not exist
2020-04-29 23:35:07.972687: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 AVX512F FMA
2020-04-29 23:35:07.979301: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2400000000 Hz
2020-04-29 23:35:07.979528: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x55c9be56bb80 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-04-29 23:35:07.979547: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
[I 23:35:20.181 LabApp] Saving file at /experiments/notebooks/many_to_one/tcn_m2o.ipynb
slurmstepd: error: *** JOB 4663510 ON tiger-h19c1n16 CANCELLED AT 2020-04-29T23:36:05 DUE TO TIME LIMIT ***
